### ISSUE TITLE
Some UI issues

### ISSUE THREAD

**Issue by Silver-IT** [okTurtles/group-income#2250]
### Problem

Down here, I will describe the issues I found while testing.
~**1. Pinned message is different from the original one.**~

![image](https://github.com/user-attachments/assets/8bc08c0f-9e2f-48ec-a833-a4563bf6e40f)

I tested with the following text.
```
You raise me up

When I am down and, oh my soul, so weary
When troubles come and my heart burdened be
Then, I am still and wait here in the silence
Until You come and sit awhile with me.
Chorus
You raise me up, so I can stand on mountains
You raise me up, to walk on stormy seas
I am strong, when I am on your shoulders
You raise me up to more than I can be
```

EDITED: Fixed in the PR #2251 

### Solution

Fix it

**Comment by Silver-IT** [comment]
~**2. Messages are not loaded**~

[Steps to reproduce]
- [Tab1] user1 creates a group
- [Tab2] user2 joins the group and creates a private channel
- [Tab2] user2 creates a message in private channel
- [Tab2] user2 copies the message link and pastes it in general chatroom
- [Tab1] user1 clicks the message link (to the one of private chatroom) and gets the alert (GOOD SO FAR)
- [Tab1] user1 copies the message link and opens it in the browser
- [Tab1] user1 is in the general chatroom, but no messages are loaded. Plus, gets the following error
![image](https://github.com/user-attachments/assets/c83f7752-cb51-4e6c-a08c-e1365c92a5cd)

EDITED: Fixed in the PR #2251

**Comment by Silver-IT** [comment]
~**3. Empty popup when the invite link is revoked**~

When I tried to revoke the general invite link, and clicked the `Action` button the empty popup is displayed.
If there is no available actions, the `Action` button should not be there I think.
![image](https://github.com/user-attachments/assets/8a2dd476-5e1f-463e-9082-ce2b8a3fb724)

EDITED: Fixed in the PR #2251

**Comment by Silver-IT** [comment]
~**4. Reactions are overflow**~

Also I think we need limit on the number of reactions or to wrap them to fix the overflow of reactions.
![image](https://github.com/user-attachments/assets/d08251cb-40ad-49fe-8565-8d44c3099250)

EDITED: This is fixed in the PR #2247.

**Comment by Silver-IT** [comment]
~**5. Overflowed reactions inside pinned messages**~

Also for the pinned messages too.
![image](https://github.com/user-attachments/assets/212266a0-3a77-473b-9394-b8f9c143e443)

EDITED: This is fixed in the PR #2247.
